src/scripts.js

```diff
@@ -170,13 +169,15 @@ getData('hydration/hydrationData').then(function(hydrationData) {
       },
       body: JSON.stringify({
         userID: user.id,
-        date: `${dateInput}`,
+        date: date,
         numOunces: ounceInput
       })
     })
+    .then(response => response.json())
+    .then(data => $('#water-consumed').text(`${data.numOunces} Ounces \n\n`))
+    .then()
     .then($('.hydration-submit').after('<p class="success-message">Data Submitted Successfully!</p>'))
     .catch();
-    $('#user-hydration-date').val('');
     $('#user-hydration-oz').val('');
```

Comment:
I think here is also where we can put an error message within a .catch that will show whenever the put in data in the wrong format and it can display like the successful message on 179l

Reply:
I added the .catch() error for sleep, and then added it for hydration (the hydration catch() is in the sleep commits)

The problem here is that if you don't put anything in the inputs at all, both the .then() and the .catch() functions run. I was planning on asking either Brittney or Khalid about this.